bot/exts/filters/filtering.py

```diff
@@ -481,7 +482,10 @@ async def _has_urls(self, text: str) -> Tuple[bool, Optional[str]]:
         for match in URL_RE.finditer(text):
             for url in domain_blacklist:
                 if url.lower() in match.group(1).lower():
```

Comment:
I'd save `url.lower()` and `match.group(1).lower()` into separate variables just because those values are used a couple of times each, but it's not a big deal here.